.github/actions/rpm/action.yml

```diff
@@ -52,7 +52,7 @@ runs:
     - name: Set RPM and SRPM
       run: |
         VERSION=$(rpm -q --qf "%{VERSION}\n" --specfile warewulf.spec)
-        GENERIC_RELEASE=$(rpm -q --qf "%{RELEASE}\n" --specfile warewulf.spec | cut -d. -f1-2)
+        GENERIC_RELEASE=$(rpm -q --qf "%{RELEASE}\n" --specfile warewulf.spec | cut -d. -f1)
         RPM=warewulf-${VERSION}-${GENERIC_RELEASE}.${{ inputs.dist }}.${{ inputs.arch }}.rpm
```

Comment:
This works for now; but be advised that if and when https://github.com/warewulf/warewulf/pull/1191 is merged, this won't pickup the `warewulf-dracut` package.

Reply:
yeah, thanks! I added uploading warewulf-dracut rpm as well in the following code